src/HttpKernel/Bundle/ResourcesProvider.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/**
+ * This file is part of Contao.
+ *
+ * Copyright (c) 2005-2015 Leo Feyer
+ *
+ * @license LGPL-3.0+
+ */
+
+namespace Contao\CoreBundle\HttpKernel\Bundle;
+
+class ResourcesProvider
```

Comment:
IMO the class should have a singular naming `ResourceProvider` no matter how many resources it provides.
I have not seen any plural naming in any symfony project for classes like this and AFAIR we have all agreed to use singular naming throughout Contao code.